monty_hall_paradox/lib/monty_hall_paradox/supervisor.ex

```diff
@@ -0,0 +1,36 @@
+defmodule Mhp.SubSupervisor do
+  use Supervisor
+
+  def start_link() do
+    {:ok, _pid} = Supervisor.start_link(__MODULE__, [], name: __MODULE__)
+  end
+
+  defp start_sim(1) do
+    {:ok, server} = Supervisor.start_child(__MODULE__, worker(Mhp.Server, [], id: 1, restart: :temporary))
+    Mhp.Server.sim_game(server)
+  end
+
+  defp start_sim(id) do
```

Comment:
I love this: let's just kick off 100,000 processes :)

My only minor quibble is the duplication between the `start_sim` functions, but is doesn't affect your grade.